ha/core/controllers/pcs/actuator_req_manager.py

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env python3
```

Comment:
This code is not pcs specific, should not be in pcs package.

Reply:
Yes, I was also thinking the same thing.
can we have this in ha/core/node ?  or a new folder altogether ?

Reply:
Moved the file to ha/util